### PR TITLE
LPS-145585 SF not reporting error on serviceReferenceUtilClassNames without `,\`

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
@ling-alan-huang  
Hi Alan,
I made some mistakes on ./source-formatter.properties where I missed typing some `./` at the end for `source.check.JavaOSGiReferenceCheck.serviceReferenceUtilClassNames`

Could you please help us see if we can change SF so that it can catch this type of error in the future for this property?

Thank you!

File: source-formatter.properties [hunk 1]
```diff
@@ -545,6 +545,7 @@
         com.liferay.portal.kernel.xmlrpc.XmlRpcUtil
         com.liferay.portal.kernel.zip.ZipReaderFactoryUtil
         com.liferay.portal.kernel.zip.ZipWriterFactoryUtil
+        com.liferay.sites.kernel.util.SitesUtil
```
Comment (julschong) [inline source-formatter.properties:548]: mistake is here

Comment (ling-alan-huang) [reply source-formatter.properties:548]: Hey @julschong 

That is an interesting issue.

I did some tests for it, I think SF is not smart enough to know how to check it.
Let me show some examples to explain why we can't add SF logic to check it.

----------------
_Example 1:_
```
key=a\
    b
```
The value of `key` is `ab`.

-------------------
_Example 2;_
```
key=a
    b
```
Oh, now we will have two keys.
the value of `key` is `a`, and the value of `b` is `empty` even we didn't add `=` for `b`.

-------------------
BTW, I use `java.util.Properties` to load properties.

Let me know what you think about it.

Comment (ling-alan-huang) [reply source-formatter.properties:548]: I'm just thinking showing examples to explain it is better than my bad English.

Comment (julschong) [reply source-formatter.properties:548]: that is very interesting :smile: I'll let Tina know tomorrow and see if we are ok

**Pull request closed without merging**